contracts/vesting/VVVVesting.sol

```diff
@@ -159,7 +159,9 @@ contract VVVVesting is VVVAuthorizationRegistryChecker {
             revert AmountIsGreaterThanWithdrawable();
         }
 
-        vestingSchedule.tokenAmountWithdrawn += _tokenAmountToWithdraw;
+        unchecked {
+            vestingSchedule.tokenAmountWithdrawn += uint128(_tokenAmountToWithdraw);
```

Comment:
Just to keep the input args simpler across the tests - there is negligible gas difference (maybe 40 gas) between the two approaches. Either works - if you prefer having the input args specified directly, can change this. It would apply in some other spots as well.